pkg/cmd/admin/router/router.go

```diff
@@ -293,6 +299,15 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 					{Type: dapi.DeploymentTriggerOnConfigChange},
 				},
 				Template: dapi.DeploymentTemplate{
+					Strategy: dapi.DeploymentStrategy{
```

Comment:
So that said, my preference would be

1.  Separate PR that sets the defaults individually by field
1.  Rebase this PR on top of that and you only need to set `UpdatePercent`